.azure-pipelines/azure-pipeline.yml

```diff
@@ -0,0 +1,37 @@
+trigger:
+- master
+
+pool:
+  vmImage: windows-latest
+
+variables:
+- group: npm-tokens
+
+steps:
+- checkout: self
+  clean: true
+
+- task: NodeTool@0
+  displayName: Use node 10
+  inputs:
+    versionSpec: "10.x"
+
+- script: npm i -g npm@6.14.12 --force
+  displayName: Use npm version 6.14.12
+
+- script: npm ci
+  displayName: Install npm-dependencies
+
+- script: npm run build
+  displayName: Build TFX CLI
+
+- powershell: |
```

Comment:
Maybe we can use this instead?
https://learn.microsoft.com/en-us/azure/devops/pipelines/artifacts/npm?view=azure-devops&tabs=yaml
As far as I remember, we already switched some repos from PS publish to NPM task.

Reply:
I think we can, but we need to get NPM service connection in `PipelineTools` project and need to investigate how the task works (because default behavior of `npm publish` is a publishing root folder to npm, but we need to publish only `_build` folder).
I think we can proceed with implemented logic and then migrate to `Npm` task. @kirill-ivlev, what do you think?

Reply:
AFAIK, to publish only build dir, it shoul be enough to add `files` property in package.json
https://docs.npmjs.com/cli/v6/configuring-npm/package-json#files

Reply:
We can switch to the NPM task later. We need to publish a new tfx-cli version now